pyorerun/biorbd_components/mesh.py

```diff
@@ -50,17 +51,19 @@ def _set_rerun_mesh3d(self):
             )
         else:
             self.__rerun_mesh = rr.Mesh3D(
-                vertex_positions=self.__mesh.vertices,
+                vertex_positions=transformed_trimesh.vertices,
                 vertex_normals=transformed_trimesh.vertex_normals,
                 vertex_colors=np.tile(self.__color, (self.__mesh.vertices.shape[0], 1)),
                 triangle_indices=self.__mesh.faces,
             )
 
     @classmethod
-    def from_file(cls, name, file_path: str, transform_callable) -> "TransformableMeshUpdater":
+    def from_file(
+        cls, name, file_path: str, transform_callable, scaling_factor: np.ndarray
+    ) -> "TransformableMeshUpdater":
         if file_path.endswith(".stl") or file_path.endswith(".STL"):
             mesh = load(file_path, file_type="stl")
-            return cls(name, mesh, transform_callable)
+            return cls(name, mesh, transform_callable, scaling_factor)
```

Comment:
Can we apply the scaling to the mesh before line 80 it would save an attribute to the class that is meant to update the position of the mesh not to scale it. Maybe no, but if yes, it needs to be applied in the `from_file`method.

Reply:
As I understood it, there were two options:
1. Scale the mesh ourselves: modify the position of the mesh vertices (which I was afraid to do if I am honest)
2. Scale the mesh during visualization at the same time as visualizing its position and orientation (I found this option easier since rerun does the hard work for us)